### ISSUE TITLE
question:is this project support webpack?

### ISSUE THREAD

**Issue by MartinRGB** [pmndrs/react-three-editor#24]

**Comment by nksaraf** [comment]
No, not yet and not sure sure what would be needed to support webpack. it needs a dev server integration

**Comment by itsdouges** [comment]
Could we start moving away from the Vite specific integrations? For example the current api server is tied directly to Vite, it could be its own standalone process that's agnostic.

**Comment by krispya** [comment]
What would that look like? Would it require an additional install?

**Comment by itsdouges** [comment]
It wouldn't we can ship the code for a node process (either as a package or module) in editor package and start it up in the plugin as a sub process.

**Comment by krispya** [comment]
I don't know much about how this changes the backend but from a DX perspective this sounds good to me. 👍

**Comment by nksaraf** [comment]
I would want to approach this in a way like this. We make things work with one setup, right now vite. This will give us a few parts. 
- Babel plugins to apply to source code when being run in the browser. 
- Transforms for modifying the code using babel.
- RPC API between the browser and any supported dev server.

With this we should be able to support any other bundler/platform. For any bundler, its better to integrate with its tooling to get efficiency gains and use whatever comes out of the box.

**Comment by netgfx** [comment]
Is there a roadmap for this feature? I would like to add this to an existing project made with NextJS. I assume this is not possible at the time?

**Comment by krispya** [comment]
I believe this project has been abandoned. I'd suggest looking at Triplex: https://github.com/try-triplex/triplex
